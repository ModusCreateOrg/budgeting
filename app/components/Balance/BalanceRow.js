// @flow
import * as React from 'react';

import styles from './style.scss';

type BalanceProps = {
  children: React.Node,
};

const BalanceRow = ({ children }: BalanceProps) => (
  <div className={styles.balanceRowContainer}>
    <div className={styles.balanceRow}>{children}</div>
  </div>
);

export default BalanceRow;
