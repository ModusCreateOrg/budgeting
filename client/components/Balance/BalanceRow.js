import React, { PropTypes } from 'react';

import styles from './style.scss';

const Balance = ({ children }) => (
  <div className={styles.balanceRowContainer}>
    <div className={styles.balanceRow}>
      {children}
    </div>
  </div>
);

Balance.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Balance;
