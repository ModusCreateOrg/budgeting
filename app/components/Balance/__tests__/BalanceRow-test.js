import React from 'react';
import renderer from 'react-test-renderer';
import BalanceRow from '../BalanceRow';

it('renders correctly', () => {
  const tree = renderer.create(<BalanceRow>test</BalanceRow>).toJSON();
  expect(tree).toMatchSnapshot();
});
