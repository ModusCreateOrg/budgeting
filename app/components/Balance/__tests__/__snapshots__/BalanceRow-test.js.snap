// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="balanceRowContainer"
>
  <div
    className="balanceRow"
  >
    test
  </div>
</div>
`;
